Weekly-Store-Sales.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Weekly Store Market Sales By Region</title><!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <style>
+    .header {
+      text-align: center;font-style: italic; color:green;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <div class="header">
+      <h1>Weekly Store Sales by Market</h1>
+      <u>Comparison of Medium to Large Markets.</u>
```

Comment:
Okay, cool. For next week's assignment, could we swap this out with some real data? Even for these simple replication projects, our goal is that you end up with a real project that you could potentially share with the world.